File: tests/expected/sys_argv.go
package main

import (
	"fmt"
	"os"
)

func main() {
	var a []string = os.Args
	var cmd string = a[0]
	if !(cmd != "") {
		panic("assert")
	}
	if len(a) > 1 {
		fmt.Printf("%v\n", a[1])
	} else {
		fmt.Printf("%v\n", "OK")
	}
}

File: tests/expected/sys_argv.nim
import os


proc main() =
  let a: seq[string] = (@[getAppFilename()] & commandLineParams())
  let cmd: string = a[0]
  assert(cmd != "")
  if len(a) > 1:
    echo a[1]
  else:
    echo "OK"

main()
